keylime/mba/elchecking/elchecker.py

```diff
@@ -76,3 +119,6 @@ def evaluate_bootlog(
             True,
         )
     return failure
+
+
+logger.info("mba.elchecking.elchecker: policy names = %s", str(policies.get_policy_names()))
```

Comment:
What's the purpose of this log line? It seems to be outside the scope of any methods.

Reply:
The log announcement runs during initialization (during the import of the `elchecker` module). 

The actual registration of policies happens during import in the elchecker design. The actual line of code is [here](https://github.com/galmasi/keylime/blob/03fb3ad3df3a52dbce37fb4b82d9e960420ec160/keylime/mba/elchecking/example.py#L404
). I won't claim to like it, but my goal throughout was to mess with the working code as little as possible. I moved things around, but I didn't change what I didn't have to.

Reply:
Yeah, I got that it happened during module init, I was more curious about why it was needed. I now see the more dynamic nature of the module being loaded and seeing the order would be useful when multiple modules are loaded. Maybe it should be a debug instead?

Reply:
Yes, I agree, "debug" is better. Will make the change.